README.md

```diff
@@ -1,104 +1,215 @@
+               path
+
+positional arguments:
+  path                  k-mer spectrum file
+
+options:
+  -h, --help            show this help message and exit
+  -f {asciidoc,double_grid,double_outline,fancy_grid,fancy_outline,github,grid,heavy_grid,heavy_outline,html,jira,latex,latex_booktabs,latex_longtable,latex_raw,mediawiki,mixed_grid,mixed_outline,moinmoin,orgtbl,outline,pipe,plain,presto,pretty,psql,rounded_grid,rounded_outline,rst,simple,simple_grid,simple_outline,textile,tsv,unsafehtml,youtrack}, --table-format {asciidoc,double_grid,double_outline,fancy_grid,fancy_outline,github,grid,heavy_grid,heavy_outline,html,jira,latex,latex_booktabs,latex_longtable,latex_raw,mediawiki,mixed_grid,mixed_outline,moinmoin,orgtbl,outline,pipe,plain,presto,pretty,psql,rounded_grid,rounded_outline,rst,simple,simple_grid,simple_outline,textile,tsv,unsafehtml,youtrack}
+                        stdout table format
+  -m STYLE, --style STYLE
+                        matplotlib style file, url, or one of available style names: ntstat.hist.default, ntstat.hist.paper, Solarize_Light2, _classic_test_patch, _mpl-gallery, _mpl-gallery-nogrid,
+                        bmh, classic, dark_background, fast, fivethirtyeight, ggplot, grayscale, seaborn-v0_8, seaborn-v0_8-bright, seaborn-v0_8-colorblind, seaborn-v0_8-dark, seaborn-v0_8-dark-
+                        palette, seaborn-v0_8-darkgrid, seaborn-v0_8-deep, seaborn-v0_8-muted, seaborn-v0_8-notebook, seaborn-v0_8-paper, seaborn-v0_8-pastel, seaborn-v0_8-poster, seaborn-v0_8-talk,
+                        seaborn-v0_8-ticks, seaborn-v0_8-white, seaborn-v0_8-whitegrid, tableau-colorblind10
+  -t TITLE, --title TITLE
+                        title to put on plot
+  --y-log, --no-y-log   plot y-axis in log scale
+  -r PLOT_RANGE, --plot-range PLOT_RANGE
+                        plot x-axis limits (inclusive) separated by a colon, i.e., use a:b to show results in the range [a, b]. 'auto' will automatically adjust the limits for better visibility.
+  -o PLOT, --plot PLOT  path to output plot
+  --probs PROBS         path to output probabilities in csv format
+  --fit-gif FIT_GIF     path to output model fit history animation
+  -c CONFIG, --config CONFIG
+                        path to differential evolution config file (json)
+```
 
-Optional arguments:
-  -h, --help     shows help message and exits
-  -v, --version  prints version information and exits
+- ntStat prints comprehensive information about the model, dataset, and *k*-mers. This information can be in various formats supported by the [tabulate](https://github.com/astanin/python-tabulate) package, set by the `-f` parameter.
+- The differential evolution algorithm's parameters can be controlled by a JSON file passed to `-c`. Refer to [scipy's documentation]() for information about these parameters. Example configuration for increasing exploration, in case ntStat fails to find a good model:
```

Comment:
Missing scipy URL?

Reply:
[D'OH](https://static.simpsonswiki.com/images/thumb/c/c3/D%27oh.jpg/200px-D%27oh.jpg), fixed in 2057c6844555100d86789b492ba3d2a316aa628a